pkgs/applications/science/electronics/kicad/i18n.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv
+, cmake
+, gettext
+, i18nSrc
+, i18nVersion
```

Comment:
Nitpick: Use `src` and `version` here, and `inherit src version` under mkDerivation.